lib/galaxy/workflow/modules.py

```diff
@@ -81,6 +81,9 @@ def get_type( self ):
     def get_name( self ):
         return self.name
 
+    def get_version( self ):
+        return None
```

Comment:
Regardless of how they are consumed - classes shouldn't in the abstract have inherited methods or attributes that don't make sense for their "type". If there are problematic if/else statements - that is an indication that perhaps a visitor pattern should be employed or functionality such as json-ification needs to move out of a manager/controller and into the module itself. 

I wouldn't renamed ``get_post_job_actions`` to ``get_post_actions`` unless we have another module that actually has a post something or other action.